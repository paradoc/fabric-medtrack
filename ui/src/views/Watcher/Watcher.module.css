.container {
  width: 100vw;
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: min-content max-content;
}

.container > header {
  font-weight: bold;
  font-size: 48px;
  padding: 36px;
}

.outlet {
  padding: 0 36px;
}