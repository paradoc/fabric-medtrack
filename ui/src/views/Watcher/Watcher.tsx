import React from 'react'
import { Outlet } from 'react-router'
import { Link } from 'react-router-dom'

import styles from './Watcher.module.css'

export default function Watcher() {
  return (
    <div className={styles.container}>
      <Link to="/watcher" className={styles.link}>
        <header>RxWatch</header>
      </Link>
      <div className={styles.outlet}>
        <Outlet />
      </div>
    </div>
  )
}
