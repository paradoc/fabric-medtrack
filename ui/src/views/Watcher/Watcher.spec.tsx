import { render } from '@testing-library/react'
import React from 'react'
import { MemoryRouter } from 'react-router'
import Watcher from './Watcher'

describe('Watcher', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <MemoryRouter>
        <Watcher />
      </MemoryRouter>
    )
    expect(container).toMatchSnapshot()
  })
})
