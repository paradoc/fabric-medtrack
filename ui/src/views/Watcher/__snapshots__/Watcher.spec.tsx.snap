// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watcher should match snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <a
      class="link"
      href="/watcher"
    >
      <header>
        RxWatch
      </header>
    </a>
    <div
      class="outlet"
    />
  </div>
</div>
`;
